app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from "react";
+import { ReactDOM } from "react-dom";
+import { getMessage } from "../Redux/reducer";
+import axios from "axios";
+import { useDispatch, useSelector } from "react-redux";
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const message = useSelector((state) => state.greeting);
+
+  async function fetchMessage() {
+    await axios.get("api/messages").then((response) => {
+      dispatch(getMessage(response.data.greeting));
+    });
+  }
+
+  useEffect(() => {
+    fetchMessage();
+  }, []);
+
+  return (
+    <>
+      <p>{message}</p>
+      <button onClick={() => fetchMessage()}>Generate new message</button>
```

Comment:
- [ ] Your app does not display anything on the screen, despite having messages in the db. When I check the dev tool I see that it returns the error below. Kindly look into that.

  ![fte](https://user-images.githubusercontent.com/54848363/196708906-8ba9b1c8-9491-4d61-a5a7-8f046f2b6dec.jpg)